<ui version="4.0" >
 <class>SMIVBase</class>
 <widget class="QWidget" name="SMIVBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>641</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" colspan="5" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QPushButton" name="pbSelectFiles" >
     <property name="text" >
      <string>Select Files...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="pbCancel" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="pbSuspend" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Suspend</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QProgressBar" name="progressBar" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="4" >
    <widget class="QPushButton" name="pbQuit" >
     <property name="text" >
      <string>Quit</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5" >
    <widget class="SMIVView" name="listView" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SMIVView</class>
   <extends>QListView</extends>
   <header>SMIVView.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
